dkp/dkp.py

```diff
@@ -267,12 +304,16 @@ def gen_scripts(
         nice_args = ""
     else:
         nice_args = " " + src_args
+
+    for env_file in info.env_files:
+        nice_args += f" --env-file '{env_file}'"
```

Comment:
Might be strange corner case with env_file with `'` in name

Reply:
AFAIK there is something like https://docs.python.org/3/library/shlex.html#shlex.quote